v2g-liberty-cards/src/charger-settings-card.ts

```diff
@@ -93,6 +94,35 @@ export class ChargerSettingsCard extends LitElement {
     `;
   }
 
+  private _isV2GlibertyLoadbalancerEnabled(): boolean {
```

Comment:
Suggestion: simplify name to `_isLoadBalancerEnabled`.
(Note the capital B.)

Reply:
OK

Reply:
OK